internal/app/machined/pkg/controllers/network/route_config.go

```diff
@@ -279,7 +280,26 @@ func (ctrl *RouteConfigController) processDevicesConfiguration(logger *zap.Logge
 			route.Family = nethelpers.FamilyInet4
 		}
 
-		route.Table = nethelpers.TableMain
+		routeTable := in.Table()
+		if routeTable == "" {
+			route.Table = nethelpers.TableMain
+		} else {
+			switch routeTable {
+			case "default":
+				route.Table = nethelpers.TableDefault
+			case "main":
+				route.Table = nethelpers.TableMain
+			case "local":
+				route.Table = nethelpers.TableLocal
+			default:
+				tableId, err := strconv.ParseUint(routeTable, 10, 32)
+				if err != nil {
+					return route, fmt.Errorf("error parsing route table id: %w", err)
+				}
+				route.Table = nethelpers.RoutingTable(uint32(tableId))
+			}
```

Comment:
@smira would this be an option for now until the machine configuration is done?

Would allow specifying routing tables either by name for the default ones or by stringified id in other cases.

Reply:
Sure let's do it. What's the approach discussing in the issue? This was only a first playground. Never the idea of pushing it through.